### PR TITLE
fix(argo-events): Added missing NATS version in values.yaml

### PR DESCRIPTION
**Pull request by rakhbari**
Discussion about changes to this new PR are contained in the previous closed PR #1725 .

Fixes issue #1724 

Signed-off-by: Ramin Akhbari <ramin@akhbari.us>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [ ] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [ ] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [ ] Any new values are backwards compatible and/or have sensible default.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

File: charts/argo-events/values.yaml [hunk 1]
```diff
@@ -66,6 +66,9 @@ configs:
       - version: latest
         natsStreamingImage: nats-streaming:latest
         metricsExporterImage: natsio/prometheus-nats-exporter:latest
+      - version: 0.22.1
```
Comment (pdrastil) [inline charts/argo-events/values.yaml:69]: One more ask - can you add following `@default` annotation to not pollute README.md with all NATs versions?

```yaml
configs:
  ## NATS event bus
  nats:
    # -- Supported versions of NATS event bus
    # @default -- See [values.yaml]
    versions:
```

Comment (rakhbari) [reply charts/argo-events/values.yaml:69]: @pdrastil Sure, no problem. But just so I'm clear, did you want me to revert the changes in `README.md` as well? Is the `README.md` something that's auto-generated by the doc gen routine?

Comment (pdrastil) [reply charts/argo-events/values.yaml:70]: @rakhbari Yes you need to regenerate README.md via `scripts/helm-docs.sh` and commit that one as well

Comment (rakhbari) [reply charts/argo-events/values.yaml:70]: @pdrastil 👍 done.

**Review by pdrastil — APPROVED** [review]

**Pull request merged**